index.html

```diff
@@ -1,48 +1,297 @@
 <!DOCTYPE html>
 <html lang="en">
 
-<head>
+  <head>
      <title> Portfolio</title>
      <meta name="viewport" content="width=device-width, initial-scale=1" />
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/twitter-bootstrap/2.3.2/css/bootstrap-combined.no-icons.min.css">
     <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
-</head>
+  </head>
 
-<body>
+ <body>
 
-<header>
-    <nav class="menu">
-    <ul class="container1">
-        <li class="logo"><a href="#">Welcome</a></li>
-    <li><i class="fa fa-bars fa-2x"></i></li>
-    </ul>
-    </nav>
+    <header>
+        <nav class="menu">
+        <ul class="container1">
+            <li class="logo"><a href="#">Welcome</a></li>
+        <li><i class="fa fa-bars fa-2x"></i></li>
+        </ul>
+        </nav>
 
-</header>
-<main class="background">
-<section class="container2">
+    </header>
+    <main class="background">
+        <section id="homepage" class="container2">
 
-    <h1 class="greeting">Hey there. I am Houssam <br> <span> I am a software developer </span> </h1>
-    <p class="presentation"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
-don’t hestiate to contact me.</p>
+            <h1 class="greeting">Hey there. I am Houssam <br> <span> I am a software developer </span> </h1>
+            <p class="presentation"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+            don’t hestiate to contact me.</p>
 
-<ul class="connect" >
-    <li class="media"><a href="#"><img alt="twitter" src="twitter.png"></a></li>
-    <li class="media"><a href="#"><img alt="linkedin" src="linkedin.png"></a></li>
-    <li class="media"><a href="#"><img alt="medium" src="medium.png"></a></li>
-    <li class="media"><a href="#"><img alt="github" src="github.png"></a></li>
-    <li class="media"><a href="#"><img alt="angellist" src="angellist.png"></a></li>
-</ul>
+            <ul class="connect" >
+                <li id="twitter" class="media"><a href="#"><img alt="twitter" src="img/twitter.png"></a></li>
+                <li class="media"><a href="#"><img alt="linkedin" src="img/linkedin.png"></a></li>
+                <li class="media"><a href="#"><img alt="medium" src="img/medium.png"></a></li>
+                <li class="media"><a href="#"><img alt="github" src="img/github.png"></a></li>
+                <li class="media"><a href="#"><img alt="angellist" src="img/angellist.png"></a></li>
+            </ul>
```

Comment:
- [ ] Let us first fix this issue, use LinkedIn icon not text
![port9](https://user-images.githubusercontent.com/55635977/129381433-01608a73-935d-4f84-8997-d4ced7f11aa1.png)